src/test/MsQuicTests.h

```diff
@@ -1177,3 +1190,7 @@ typedef struct {
     // QUIC_RUN_CUSTOM_CERT_VALIDATION
 
 #define QUIC_MAX_IOCTL_FUNC_CODE 110
+
+#define IOCTL_QUIC_RELIABLE_RESET_NEGOTIATION \
+    QUIC_CTL_CODE(111, METHOD_BUFFERED, FILE_WRITE_DATA)
+    // QUIC_RUN_RELIABLE_RESET_NEGOTIATION
```

Comment:
Add this before `QUIC_MAX_IOCTL_FUNC_CODE` and make sure to increment `QUIC_MAX_IOCTL_FUNC_CODE` to `111`. Also the comment that follows is the argument type (i.e. struct) for the IOCTL.

Reply:
You will also have to update `QUIC_IOCTL_BUFFER_SIZES` in control.cpp with the size.

Reply:
Sounds good, interesting how we implement a hash table using an array.